cumulus/parachains/runtimes/collectives/collectives-westend/src/weights/xcm/mod.rs

```diff
@@ -262,7 +262,8 @@ impl<Call> XcmWeightInfo<Call> for CollectivesWestendXcmWeight<Call> {
 		XcmGeneric::<Runtime>::clear_topic()
 	}
 	fn alias_origin(_: &Location) -> Weight {
-		XcmGeneric::<Runtime>::alias_origin()
+		// XCM Executor does not currently support alias origin operations
+		Weight::MAX
```

Comment:
why?

Reply:
my mistake, I copied this PR from fellows [PR](https://github.com/polkadot-fellows/runtimes/pull/547) where we don't allow `Aliasers` yet, fixed: https://github.com/paritytech/polkadot-sdk/pull/7215/commits/1a8ceab7bb71bd93e10a0ccf207ddf40c9581265